src/prefect/tasks/control_flow/conditional.py

```diff
@@ -81,6 +81,8 @@ def b_branch():
         - cases (Dict[Any, Task]): a dict representing the "case" statements of the switch.
             The value of the `condition` task will be compared to the keys of this dict, and
             the matching task will be executed.
+        - mapped (bool, optional): Whether the results of these tasks should be mapped over
```

Comment:
Sorry, when I commented on the `merge` docstring, I should have better indicated that request also applied to `switch` and `ifelse` - could you update those docstrings as well?

Reply:
On me. All sorted and thanks again.